From 02c0bbe60ea7b7481c5fafe1af3bc6f0d5b69994 Mon Sep 17 00:00:00 2001
From: Heinrich Toews <heinrich.toews@wago.com>
Date: Mon, 14 Nov 2016 15:21:37 +0000
Subject: [PATCH 042/292] add software license correction for GPLv2

add further software license correction for GPLv2

 Conflicts:
	arch/arm/mach-omap2/pfcxxx-boot-table.c
	include/linux/pfc-prio.h
	include/linux/pfcxxx-boot-table.h
	include/linux/wsysinit-devconf.h
	include/linux/wsysinit-prodcution.h
	kernel/irq/pfc-prio-table.c
---
 arch/arm/boot/dts/wago-pfc100-cmtests.dtsi   |  8 +++++++
 arch/arm/boot/dts/wago-pfc100-dip.dtsi       |  8 +++++++
 arch/arm/boot/dts/wago-pfc100-uart1.dtsi     |  8 +++++++
 arch/arm/mach-omap2/wsysinit-boot-table.c    | 17 +++++++++++++++
 drivers/net/ethernet/micrel/ksz8863.c        | 24 +++++++++------------
 drivers/spi/spi-kbus.c                       | 21 ++++++++++++++++++-
 drivers/tty/serial/pfc-modbus-rtu.c          | 31 ++++++++++++++++------------
 include/linux/platform_data/pfc-modbus-rtu.h | 26 +++++++++++------------
 include/linux/wsysinit-boot-table.h          | 18 ++++++++++++++++
 include/linux/wsysinit-devconf.h             | 18 ++++++++++++++++
 include/linux/wsysinit-prio.h                | 18 ++++++++++++++++
 include/linux/wsysinit-prodcution.h          | 18 ++++++++++++++++
 include/misc/wago-tests.h                    | 18 ++++++++++++++++
 include/trace/events/dsa.h                   | 18 ++++++++++++++++
 include/trace/events/pxc.h                   | 18 ++++++++++++++++
 kernel/irq/wsysinit-prio-table.c             | 18 ++++++++++++++++
 16 files changed, 246 insertions(+), 41 deletions(-)

diff --git a/arch/arm/boot/dts/wago-pfc100-cmtests.dtsi b/arch/arm/boot/dts/wago-pfc100-cmtests.dtsi
index 3fd7348..f451537 100644
--- a/arch/arm/boot/dts/wago-pfc100-cmtests.dtsi
+++ b/arch/arm/boot/dts/wago-pfc100-cmtests.dtsi
@@ -1,3 +1,11 @@
+/*
+ * Copyright (C) 2015 WAGO Kontakttechnik GmbH & Co. KG - http://global.wago.com/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
 / {
 	cmtests {
 		compatible = "wago,testenv";
diff --git a/arch/arm/boot/dts/wago-pfc100-dip.dtsi b/arch/arm/boot/dts/wago-pfc100-dip.dtsi
index debff5af..803398b 100644
--- a/arch/arm/boot/dts/wago-pfc100-dip.dtsi
+++ b/arch/arm/boot/dts/wago-pfc100-dip.dtsi
@@ -1,3 +1,11 @@
+/*
+ * Copyright (C) 2015 WAGO Kontakttechnik GmbH & Co. KG - http://global.wago.com/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
 &am33xx_pinmux {
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart1_dip_pins>;
diff --git a/arch/arm/boot/dts/wago-pfc100-uart1.dtsi b/arch/arm/boot/dts/wago-pfc100-uart1.dtsi
index 313174b..1837290 100644
--- a/arch/arm/boot/dts/wago-pfc100-uart1.dtsi
+++ b/arch/arm/boot/dts/wago-pfc100-uart1.dtsi
@@ -1,3 +1,11 @@
+/*
+ * Copyright (C) 2015 WAGO Kontakttechnik GmbH & Co. KG - http://global.wago.com/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
 &am33xx_pinmux {
 	uart1_rs_pins: pinmux_uart1_rs_pins { /* optional uart1 rs232/485 */
 		pinctrl-single,pins = <
diff --git a/arch/arm/mach-omap2/wsysinit-boot-table.c b/arch/arm/mach-omap2/wsysinit-boot-table.c
index 96e0eab..56738e5 100644
--- a/arch/arm/mach-omap2/wsysinit-boot-table.c
+++ b/arch/arm/mach-omap2/wsysinit-boot-table.c
@@ -1,3 +1,20 @@
+/*
+ * Copyright (C) 2014 Wago Kontakttechnik GmbH
+ *
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
 #include <linux/init.h>
 #include <linux/module.h>
 #include <linux/bug.h>
diff --git a/drivers/net/ethernet/micrel/ksz8863.c b/drivers/net/ethernet/micrel/ksz8863.c
index 5f9a235..24e8dae 100644
--- a/drivers/net/ethernet/micrel/ksz8863.c
+++ b/drivers/net/ethernet/micrel/ksz8863.c
@@ -1,25 +1,21 @@
 /*
- * WAGO Rocket Local Bus Tripple Buffer Core driver
+ * ksz8863 Switch Reset Driver
  *
  * Copyright (C) 2014 Wago Kontakttechnik GmbH
- *	Santosh Shilimkar <santosh.shilimkar@ti.com>
- *	Sricharan <r.sricharan@ti.com>
  *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
  *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307
- * USA
  */
+
 #undef DEBUG
 #include <linux/module.h>
 #include <linux/kernel.h>
diff --git a/drivers/spi/spi-kbus.c b/drivers/spi/spi-kbus.c
index 1a2e333..7a44eef 100644
--- a/drivers/spi/spi-kbus.c
+++ b/drivers/spi/spi-kbus.c
@@ -1,3 +1,22 @@
+/*
+ * Wago KBUS SPI Driver
+ *
+ * Copyright (C) 2013 Wago Kontakttechnik GmbH
+ *
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
 #include <linux/version.h>
 #include <linux/init.h>
 #include <linux/uaccess.h>
@@ -1239,4 +1258,4 @@ module_exit(kbus_exit);
 
 MODULE_DESCRIPTION("WAGO KBUS SPI Driver");
 MODULE_AUTHOR("Heinrich Toews <heinrich.toews@wago.com>");
-MODULE_LICENSE("GPL");
+MODULE_LICENSE("GPL v2");
diff --git a/drivers/tty/serial/pfc-modbus-rtu.c b/drivers/tty/serial/pfc-modbus-rtu.c
index 0ef377c..e17e58b 100644
--- a/drivers/tty/serial/pfc-modbus-rtu.c
+++ b/drivers/tty/serial/pfc-modbus-rtu.c
@@ -1,26 +1,28 @@
-/*------------------------------------------------------------------------------
- * Copyright (c) WAGO Kontakttechnik GmbH & Co. KG
+/*
+ * WAGO Serial Modbus RTU Driver
  *
- * PROPRIETARY RIGHTS are involved in the subject matter of this material. All
- * manufacturing, reproduction, use and sales rights pertaining to this
- * subject matter are governed by the license agreement. The recipient of this
- * software implicitly accepts the terms of the license.
- *------------------------------------------------------------------------------
- *------------------------------------------------------------------------------
- *   \file     pfc-modbus-rtu.c
+ * Copyright (C) 2013 Wago Kontakttechnik GmbH
  *
- *   \version  0.1
+ * Author: Timur Celik <timur.celik@wago.com>
  *
- *   \brief    <short description of the file contents>
  *
- *   \author   Timur Celik : WAGO Kontakttechnik GmbH & Co. KG
- *------------------------------------------------------------------------------
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
  */
+
 #undef DEBUG
 
 #ifdef CONFIG_SERIAL_OMAP_MODBUS
 
 //#include <linux/platform_data/serial-omap.h>
+#include <linux/module.h>
 #include <linux/platform_data/pfc-modbus-rtu.h>
 #include "omap-serial-rtu.h"
 
@@ -482,4 +484,7 @@ void modbus_omap_shutdown(struct uart_omap_port *up)
 {
 };
 
+MODULE_AUTHOR("Timur Celik <timur.celik@wago.com>");
+MODULE_DESCRIPTION("KSZ8863 Switch Reset Driver");
+MODULE_LICENSE("GPL v2");
 #endif
diff --git a/include/linux/platform_data/pfc-modbus-rtu.h b/include/linux/platform_data/pfc-modbus-rtu.h
index f45e3c7..68c2fed 100644
--- a/include/linux/platform_data/pfc-modbus-rtu.h
+++ b/include/linux/platform_data/pfc-modbus-rtu.h
@@ -1,20 +1,20 @@
-/*------------------------------------------------------------------------------
- * Copyright (c) WAGO Kontakttechnik GmbH & Co. KG
+/*
+ * WAGO Serial Modbus RTU Driver
  *
- * PROPRIETARY RIGHTS are involved in the subject matter of this material. All
- * manufacturing, reproduction, use and sales rights pertaining to this
- * subject matter are governed by the license agreement. The recipient of this
- * software implicitly accepts the terms of the license.
- *------------------------------------------------------------------------------
- *------------------------------------------------------------------------------
- *   \file     pfc-modbus-rtu.h
+ * Copyright (C) 2013 Wago Kontakttechnik GmbH
  *
- *   \version  0.1
+ * Author: Timur Celik <timur.celik@wago.com>
  *
- *   \brief    <short description of the file contents>
  *
- *   \author   Timur Celik : WAGO Kontakttechnik GmbH & Co. KG
- *------------------------------------------------------------------------------
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
  */
 
 #ifndef PFC_MODBUS_RTU_H_
diff --git a/include/linux/wsysinit-boot-table.h b/include/linux/wsysinit-boot-table.h
index c5ec9fa..aae19b1 100644
--- a/include/linux/wsysinit-boot-table.h
+++ b/include/linux/wsysinit-boot-table.h
@@ -1,3 +1,21 @@
+/*
+ *
+ * Copyright (C) 2014 Wago Kontakttechnik GmbH
+ *
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
 #ifndef _WSYSINIT_BOOT_TABLE_H
 #define _WSYSINIT_BOOT_TABLE_H
 
diff --git a/include/linux/wsysinit-devconf.h b/include/linux/wsysinit-devconf.h
index 50cf3a7..ac4f5f9 100644
--- a/include/linux/wsysinit-devconf.h
+++ b/include/linux/wsysinit-devconf.h
@@ -2,6 +2,24 @@
 #define _WSYSINIT_DEVCONF_H
 
 /*
+ *
+ * Copyright (C) 2014 Wago Kontakttechnik GmbH
+ *
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
+/*
  * The devconf variable is a 16bit value.
  * Data is stored in little endian format.
  *
diff --git a/include/linux/wsysinit-prio.h b/include/linux/wsysinit-prio.h
index b28f288..9689335 100644
--- a/include/linux/wsysinit-prio.h
+++ b/include/linux/wsysinit-prio.h
@@ -1,3 +1,21 @@
+/*
+ *
+ * Copyright (C) 2014 Wago Kontakttechnik GmbH
+ *
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
 #ifndef __WSYSINIT_PRIO_H
 #define __WSYSINIT_PRIO_H
 
diff --git a/include/linux/wsysinit-prodcution.h b/include/linux/wsysinit-prodcution.h
index 742dfb7..0e03229 100644
--- a/include/linux/wsysinit-prodcution.h
+++ b/include/linux/wsysinit-prodcution.h
@@ -2,6 +2,24 @@
 #define _WSYSINIT_PRODUCTION_H
 
 /*
+ *
+ * Copyright (C) 2014 Wago Kontakttechnik GmbH
+ *
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
+/*
  * TERMINOLOGY:
  *    -            firmware: the complete software (bootloader, linux, rootfs, codesys runtime, ...)
  *                           that will run later on the product (size: about 18MB)
diff --git a/include/misc/wago-tests.h b/include/misc/wago-tests.h
index a57bc28..de1ac09 100644
--- a/include/misc/wago-tests.h
+++ b/include/misc/wago-tests.h
@@ -1,3 +1,21 @@
+/*
+ *
+ * Copyright (C) 2014 Wago Kontakttechnik GmbH
+ *
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
 #ifndef _WAGO_TESTS_H_
 #define _WAGO_TESTS_H_
 
diff --git a/include/trace/events/dsa.h b/include/trace/events/dsa.h
index 958a9f0..b1230f5 100644
--- a/include/trace/events/dsa.h
+++ b/include/trace/events/dsa.h
@@ -1,3 +1,21 @@
+/*
+ *
+ * Copyright (C) 2015 Wago Kontakttechnik GmbH
+ *
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
 #undef TRACE_SYSTEM
 #define TRACE_SYSTEM dsa
 
diff --git a/include/trace/events/pxc.h b/include/trace/events/pxc.h
index 93beebc..45b37bb 100644
--- a/include/trace/events/pxc.h
+++ b/include/trace/events/pxc.h
@@ -1,3 +1,21 @@
+/*
+ *
+ * Copyright (C) 2015 Wago Kontakttechnik GmbH
+ *
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
 #undef TRACE_SYSTEM
 #define TRACE_SYSTEM pxc
 
diff --git a/kernel/irq/wsysinit-prio-table.c b/kernel/irq/wsysinit-prio-table.c
index d87d489..f754f0a 100644
--- a/kernel/irq/wsysinit-prio-table.c
+++ b/kernel/irq/wsysinit-prio-table.c
@@ -1,3 +1,21 @@
+/*
+ *
+ * Copyright (C) 2014 Wago Kontakttechnik GmbH
+ *
+ * Author: Heinrich Toews <heinrich.toews@wago.com>
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
 #include <linux/kernel.h>
 #include <linux/module.h>
 #include <linux/device.h>
-- 
2.7.4

